cli/status.go

```diff
@@ -240,6 +263,20 @@ func clusterStatus(cn *pbm.PBM, uri string) (fmt.Stringer, error) {
 	return ret, nil
 }
 
+func replSetGetConfig(ctx context.Context, m *mongo.Client) (*pbm.RSConfig, error) {
+	res := m.Database("admin").RunCommand(ctx, bson.D{{"replSetGetConfig", 1}})
+	if err := res.Err(); err != nil {
+		return nil, errors.WithMessage(err, "run command")
+	}
+
+	val := struct{ Config *pbm.RSConfig }{}
+	if err := res.Decode(&val); err != nil {
+		return nil, errors.WithMessage(err, "decode")
+	}
+
+	return val.Config, nil
+}
```

Comment:
we have a similar method in pbm/node.go - `Node.GetRSconf()`. Let's move this func there and `Node.GetRSconf()` will call it